01week/rockPaperScissors.js

```diff
@@ -7,17 +7,62 @@ const rl = readline.createInterface({
   output: process.stdout
 });
 
+/*
+Define a function named rockPaperScissors()
+rockPaperScissors() will accept two parameters: hand1, hand2
+Using conditional statements, compare the two hands to one another
+ */
+const rockPaperScissors = (hand1, hand2) => {
+
+    //Convert user input to lowercase and trim whitespace
+    hand1 = hand1.toLowerCase();
+    hand1 = hand1.trim();
+    hand2 = hand2.toLowerCase();
+    hand2 = hand2.trim();
+
```

Comment:
great use of methods! To refactor this code you can actually chain methods , and can cut this down to two lines using:  player1 = hand1.toLowerCase().trim();